libs/core/src/lib/menu/menu.component.ts

```diff
@@ -129,6 +129,10 @@ export class MenuComponent implements MenuInterface, AfterContentInit, AfterView
     @Output()
     readonly activePath: EventEmitter<MenuItemComponent[]> = new EventEmitter<MenuItemComponent[]>();
 
+    /** Emits array of active menu items */
+    @Output()
+    readonly selected: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
Should emit `MenuItem`

Reply:
fixed

Reply:
You should also avoid explicit typing when property is declared and initialized at the same time.

Reply:
fixed